pubspec.yaml

```diff
@@ -25,6 +25,8 @@ dependencies:
     path: api/packages/player_repository
   app_ui:
     path: packages/app_ui
+  flutter_screenutil: ^5.9.0
```

Comment:
It's better to not depend on third party packages for responsiveness and adaptiveness